models/cost_per_query.sql

```diff
@@ -159,7 +159,7 @@ select
     all_queries.end_time,
     all_queries.execution_start_time,
     all_queries.compute_cost,
-    (all_queries.credits_used_cloud_services / credits_billed_daily.daily_credits_used_cloud_services * credits_billed_daily.daily_billable_cloud_services) * daily_rates.effective_rate as cloud_services_cost,
+    (all_queries.credits_used_cloud_services / nullif(credits_billed_daily.daily_credits_used_cloud_services, 0) * credits_billed_daily.daily_billable_cloud_services) * daily_rates.effective_rate as cloud_services_cost,
```

Comment:
```suggestion
    (div0(all_queries.credits_used_cloud_services, credits_billed_daily.daily_credits_used_cloud_services) * credits_billed_daily.daily_billable_cloud_services) * daily_rates.effective_rate as cloud_services_cost,
```

good catch! i think we want `div0` though? with `nullif` the column will become null instead of 0.

Reply:
100% - that's a good plan.